assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidgetGA4.js

```diff
@@ -24,40 +24,76 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, _x } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { ADSENSE_GA4_TOP_EARNING_PAGES_NOTICE_DISMISSED_ITEM_KEY as DISMISSED_KEY } from '../../constants';
 import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import {
+	MODULES_ANALYTICS,
+	DATE_RANGE_OFFSET,
+} from '../../../analytics/datastore/constants';
+import { generateDateRangeArgs } from '../../../analytics/util/report-date-range-args';
 import whenActive from '../../../../util/when-active';
+import SourceLink from '../../../../components/SourceLink';
 import SettingsNotice from '../../../../components/SettingsNotice';
+import useViewOnly from '../../../../hooks/useViewOnly';
 const { useSelect } = Data;
 
-function DashboardTopEarningPagesWidgetGA4( { WidgetNull } ) {
+function DashboardTopEarningPagesWidgetGA4( { WidgetNull, Widget } ) {
```

Comment:
Looks like this component is missing prop type for the `WidgetNull` property. Could you please add it?